assets/src/scss/layout/_navbar.scss

```diff
@@ -272,3 +272,4 @@ a.nav-link {
 @import "navbar/site-logo";
 @import "navbar/languages";
 @import "navbar/search";
+@import "navbar/country-selector-banner";
```

Comment:
Since we only would actually use this on a single site, we should probably avoid loading the unused CSS on all the other sites.

Reply:
Would it make things easier if we move this code [here](https://github.com/greenpeace/planet4-child-theme-international)?

We can only keep the `{% include 'country_selector_banner.twig' %}` here. It should failed silently if the template is missing.

Reply:
I guess we could do that yes, if we're sure it won't cause any crash! @Inwerpsel what do you think?

Reply:
Sounds good :+1: We could also add the template to the child theme if we include an empty one in master theme and override it with a non empty one in the international child theme.